src/schema/metadata/MEGCoordinateSystem.yaml

```diff
@@ -0,0 +1,13 @@
+---
+key_name: MEGCoordinateSystem
+description: |
+  Defines the coordinate system for the MEG sensors.
+  See [Appendix VIII](/99-appendices/08-coordinate-systems.html)
+  for a list of restricted keywords for coordinate systems.
+  If `"Other"`, provide definition of the coordinate system in
+  `MEGCoordinateSystemDescription`.
+anyOf:
+  - $ref: _MEGCoordSys.yaml
+  - $ref: _EEGCoordSys.yaml
+  - $ref: _StandardTemplateCoordSys.yaml
+  - $ref: _StandardTemplateDeprecatedCoordSys.yaml
```

Comment:
This currently renders as "string or string or string or string". Not sure how best to clean it up.

Reply:
I tried rendering it with the enum typed fields joined with " or ", unfortunately the resulting list seems unwieldy to put directly into the table. This also wouldn't make sense for other fields like GeneticLevel where the enumerations are listed directly in the description.

Is there any current situation where removing duplicates of  what `_resolve_metadata_type` returns when generating the string for anyOf would cause a problem?

Reply:
Currently it's just a single single that's appended to, so it's difficult to identify and remove duplicates. I could reorganize it to a list of lists or something that would be easier to identify and remove duplicates though.